src/plans/proximal_gradient_plan.jl

```diff
@@ -517,7 +517,6 @@ function (s::ProximalGradientMethodBacktrackingStepsize)(
         # Reduce step size
         λ *= s.contraction_factor
     end
-    @warn "Backtracking stopped because the stepsize fell below the threshold $(s.stop_when_stepsize_less)."
     return λ
```

Comment:
This line now seems to never be reached says code cov, is this true or is code cov incorrect?